inventory/byo/hosts.ose.example

```diff
@@ -167,6 +167,13 @@ openshift_release=v3.6
 # modify image streams to point at that registry by setting the following to true
 #openshift_examples_modify_imagestreams=true
 
+# If oreg_url points to a registry requiring authentication, provide the following:
+oreg_auth_user=some_user
+oreg_auth_password='my-pass'
```

Comment:
Lets comment these and make it clear that you *must* set oreg_url for this to work, otherwise we don't have a registry host to log into. otherwise LGTM.